tests/Api/ReportsTest.php

```diff
@@ -38,7 +37,6 @@ public function testGetCustom()
 
         $this->assertErrors($response);
         $this->assertGreaterThanOrEqual(0, $response['totalResults']);
-        $this->assertGreaterThanOrEqual(0, count($response['totalResults']));
```

Comment:
Why was this removed? The `totalResutls` param is not in the response anymore?

Reply:
It's still there, but in the line above it, it's already checked whether its value is greater than or equal to 0. Since it's an integer, it's throwing a warning in PHP 7.2+ as integers are non-countable (https://www.php.net/manual/en/migration72.incompatible.php#migration72.incompatible.warn-on-non-countable-types)